docs/content/deploy.md

```diff
@@ -0,0 +1,121 @@
+**Warning:** The [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
+
+-----
+## Deploying your game
+
+When you want to try out your game, you need to run a deployment of the game. This means launching SpatialOS itself. SpatialOS sets up the game world and starts up the workers needed to run the game world. Once the deployment is running, you can connect clients to it in order to play the game. You can run a deployment on your development machine (a "local deployment" or in the cloud (a "cloud deployment"). 
```

Comment:
`...starts up the workers need to run the game world...` given we show the opposite in the Playground should we still note this? 

Maybe "optionally starts up workers"

Reply:
also renamed workers to server-workers to stay consistent with the terminology